### PR TITLE
[patch] Fix app deletion

### PR DESCRIPTION
**Pull request by padmankosalaram**
MASCORE-190

### CODE REVIEW

File: image/cli/mascli/functions/gitops_utils [hunk 1]
```diff
@@ -282,11 +283,9 @@ function check_argo_app_synced() {
       echo "Sync Status is $SYNC_STATUS, Waiting 30s before checking status again"
       ((COUNT++))
       if ! (( $COUNT % 10 )) ; then
-        echo "Restart application-controller and try resync"
-        oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
-        echo 'Waiting for rollout of application-controller to complete'
-        oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops
-
+        if [ -n "${NAMESPACE}" ]; then
+          oc adm policy add-role-to-user admin system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller -n ${NAMESPACE}
```
Comment (whitfiea) [inline image/cli/mascli/functions/gitops_utils:288]: Can we add a comment just to say why we are going this. Just so when we look back 6 months we can understand why we added this call

File: image/cli/mascli/functions/gitops_init_suite_config [hunk 2]
```diff
@@ -286,6 +286,12 @@ function gitops_init_suite_config() {
   # Commit and push to github target repo
   # ---------------------------------------------------------------------------
   if [ "$GITHUB_PUSH" == "true" ]; then
+    echo
+    # echo_h2 "Restart application-controller ..."
+    # oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
+    # echo_h2 'Waiting for rollout of application-controller to complete'
+    # oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops
```
Comment (whitfiea) [inline image/cli/mascli/functions/gitops_init_suite_config]: should we remove these commented out lines?

File: image/cli/mascli/functions/gitops_init_suite [hunk 3]
```diff
@@ -330,6 +330,12 @@ function gitops_init_suite() {
   # Commit and push to github target repo
   # ---------------------------------------------------------------------------
   if [ "$GITHUB_PUSH" == "true" ]; then
+    echo
+    # echo_h2 "Restart application-controller ..."
+    # oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
+    # echo_h2 'Waiting for rollout of application-controller to complete'
+    # oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops
```
Comment (whitfiea) [inline image/cli/mascli/functions/gitops_init_suite]: should we remove these commented out lines?

File: image/cli/mascli/functions/gitops_init_cis_compliance [hunk 4]
```diff
@@ -178,6 +178,12 @@ function gitops_init_cis_compliance() {
   # Commit and push to github target repo
   # ---------------------------------------------------------------------------
   if [ "$GITHUB_PUSH" == "true" ]; then
+    echo
+    # echo_h2 "Restart application-controller ..."
+    # oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
+    # echo_h2 'Waiting for rollout of application-controller to complete'
+    # oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops
```
Comment (whitfiea) [inline image/cli/mascli/functions/gitops_init_cis_compliance]: remove these commented out lines?

File: image/cli/mascli/functions/gitops_bootstrap [hunk 5]
```diff
@@ -213,11 +213,11 @@ function gitops_bootstrap() {
...
   # 8. For some reason the application-controller needs restarting to pickup the RBAC changes
-  echo 'Restart application-controller'
-  oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
+  # echo 'Restart application-controller'
+  # oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
   
-  echo 'Waiting for rollout of application-controller to complete'
-  oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops  
+  # echo 'Waiting for rollout of application-controller to complete'
+  # oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops
```
Comment (whitfiea) [inline image/cli/mascli/functions/gitops_bootstrap]: Remove these if we don't plan to do it anymore

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: image/cli/mascli/functions/gitops_utils (hunk 1 above)
Comment (padmankosalaram) [reply image/cli/mascli/functions/gitops_utils:288]: Added the comment -
- Fix for Issue : Cannot sync application in OpenShift GitOps. https://access.redhat.com/solutions/6331341

File: image/cli/mascli/functions/gitops_init_suite_config (hunk 2 above)
Comment (padmankosalaram) [reply image/cli/mascli/functions/gitops_init_suite_config]: Removed the comments

File: image/cli/mascli/functions/gitops_init_cis_compliance (hunk 4 above)
Comment (padmankosalaram) [reply image/cli/mascli/functions/gitops_init_cis_compliance]: Removed the commented out line

File: image/cli/mascli/functions/gitops_bootstrap (hunk 5 above)
Comment (padmankosalaram) [reply image/cli/mascli/functions/gitops_bootstrap]: Removed the commented out lines.  We will monitor the fix (adding oc adm policy add-role-to-user admin)

**Review by whitfiea — APPROVED** [review]

**Pull request merged**